main.py

```diff
@@ -38,7 +39,22 @@ def main():
         config_file = networkconfig[conf]
         with tf.Session() as sess:
             builder = ModelBuilder(config_file, sess)
+            builder.add_input_layer()
+
+            # Add a number of hidden layers
```

Comment:
Why is this done in main.py? Shouldn't it be done in the build function of the model builder?